src/platform/web/theming/DerivedVariables.ts

```diff
@@ -98,8 +98,8 @@ export class DerivedVariables {
     }
 }
 
-import pkg from "off-color";
-const {offColor} = pkg;
+import * as pkg from "off-color";
+const offColor = pkg.offColor ?? pkg.default.offColor;
```

Comment:
I wonder if this would fix the tsc error?
```suggestion
const offColor = pkg.offColor ?? pkg.default?.offColor;
```